qcodes/tests/test_abstract_instrument.py

```diff
@@ -172,6 +176,7 @@ def test_subsub():
     assert instance.call_count == 1
 
 
+@pytest.mark.skip("tests unimplemented feature")
```

Comment:
shall we do xfail instead? so that these stay runnable during other development work?
```suggestion
@pytest.mark.xfail("tests unimplemented feature")
```

Reply:
We could do that but it would require a significant amount of work to make these test run cleanly since they now leak partially created instruments. I do not think it is worth the effort.

Reply:
fair